src/integer/z/arithmetic/div.rs

```diff
@@ -299,14 +299,14 @@ mod test_div_ceil {
         value.div_ceil(Z::from(10));
         value.div_ceil(Modulus::from(10));
 
-        value.div_ceil(&3_u8);
-        value.div_ceil(&3_u16);
-        value.div_ceil(&3_u32);
-        value.div_ceil(&3_u64);
-        value.div_ceil(&3_i8);
-        value.div_ceil(&3_i16);
-        value.div_ceil(&3_i32);
-        value.div_ceil(&3_i64);
+        value.div_ceil(3_u8);
+        value.div_ceil(3_u16);
+        value.div_ceil(3_u32);
+        value.div_ceil(3_u64);
+        value.div_ceil(3_i8);
+        value.div_ceil(3_i16);
+        value.div_ceil(3_i32);
+        value.div_ceil(3_i64);
```

Comment:
with these changes the same code is included twice (see lines above)

I would either say, leave them here for consistency and add an allow clippy before, or remove them.
Up to you imo